src/linpred.jl

```diff
@@ -1,4 +1,9 @@
 ## Return the linear predictor vector
+"""
+    linpred!(out, p::LinPred, f::Real=1.)
+
+Linear predictor inplace.
```

Comment:
The step factor `f` determines the parameter value at which the linear predictor is evaluated in an iterative algorithm like IRLS.  The current coefficient vector is `beta0` and the proposed increment is `delbeta`.  The linear predictor is evaluated at `beta0 + f .* delbeta`